feature-flags/src/lib.rs

```diff
@@ -3,5 +3,14 @@ pub mod config;
 pub mod redis;
 pub mod router;
 pub mod server;
+pub mod team;
 pub mod v0_endpoint;
 pub mod v0_request;
+
+// Test modules don't need to be compiled with main binary
+// #[cfg(test)]
+// TODO: To use in integration tests, we need to compile with binary
```

Comment:
looking for suggestions here

Reply:
Probably not worth adding too much complexity for the time being. Once we have enough helpers that are generic enough, they could be moved to a separate common crate.

Reply:
works, sounds good!